### DISCUSSION TITLE
Mute people in the new beta feed

### DISCUSSION THREAD

**Discussion by PurHur** [community/community#27601] [Other Feature Feedback, Questions, & Ideas]
There is one guy i "contributed with" aka he answered in the same PR as i.

He is making 1000 Lists of Repos and my whole feed is unuseable because its full of these bad lists.

**Comment by jfullstackdev** [comment]
try blocking him, there was one similar situation posted, although not very sure, since it is in beta, whether it will be removed from that but it should be the behavior

**Reply by PurHur** [reply]
I dont want to block him i just dont care about his lists

**Reply by jfullstackdev** [reply]
yes, that's the request, but as for now, so you will not be annoyed anymore, you can do that blocking first while waiting for that mute feature to be implemented

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by cavo789** [reply]
@jdevstatic Thanks for the different filtering options but ... it's not a "mute" one? This is not because we've interacted with someone years ago that we're interested by his different actions. We were just interested by a given repo at a given time period.

Like others, I would really appreciate a mute option.

Right now, the feed is just unusable to me, too much noise...

Thanks

**Reply by jfullstackdev** [reply]
yup, they are working on it, much like `I don't want to hear from this user`

there will be further updates
